Wrangler/__init__.py

```diff
@@ -0,0 +1,41 @@
+import sys
+from .Linki import Linki
+from .Network import Network
+from .NetworkException import NetworkException
+from .PNRLink import PNRLink
+from .Supplink import Supplink
+##try:
+##    from .TransitAssignmentData import TransitAssignmentData
+##except:
+##    print "Unexpected error:", sys.exc_info()
+##    from .transitAssignmentData import TransitAssignmentData
+from .TransitAssignmentData import TransitAssignmentData ##
+from .TransitCapacity import TransitCapacity
+from .TransitLine import TransitLine
+from .TransitLink import TransitLink
+from .TransitNetwork import TransitNetwork
+from .TransitParser import TransitParser
+from .HighwayNetwork import HighwayNetwork
+from .Logger import setupLogging, WranglerLogger
+from .Node import Node
+from .HwySpecsRTP import HwySpecsRTP
+
+
+__all__ = ['NetworkException', 'setupLogging', 'WranglerLogger',
+           'Network', 'TransitAssignmentData', 'TransitNetwork', 'TransitLine', 'TransitParser',
+           'Node', 'TransitLink', 'Linki', 'PNRLink', 'Supplink', 'HighwayNetwork', 'HwySpecsRTP',
+           'TransitCapacity',
+]
+
+
+if __name__ == '__main__':
```

Comment:
I know you didn't write this @sdrewc , but eventually it would be good to create some tests with example data...